editor/js/editor-libs/telemetry.js

```diff
@@ -0,0 +1,68 @@
+import { postParentMessage } from "./events.js";
+
+const ACTION_COUNTS_KEY = "action-counts";
+
+function readActionsCounts() {
+  try {
+    const value = JSON.parse(localStorage.getItem(ACTION_COUNTS_KEY));
+    if (value && value.href === window.location.href && value.counts) {
+      return value.counts;
+    }
+  } catch (e) {
+    window.console.warn("Unable to read action counts from localStorage", e);
+  }
+
+  return {};
+}
+
+function persistActionCounts(counts) {
+  try {
+    localStorage.setItem(
+      ACTION_COUNTS_KEY,
+      JSON.stringify({
+        href: window.location.href,
+        counts,
+      })
+    );
+  } catch (e) {
+    window.console.warn("Unable to write action counts to localStorage", e);
+  }
+}
+
+const actionCounts = readActionsCounts();
+let lastAction = null;
+
+/**
+ * @param {string} key
+ * @param {boolean} once
```

Comment:
This comment is not up to date. It would be good to write a sentence about what this function does because the rest of the project is quite well documented.

Reply:
Resolved via 67c81fba4c99984d35ed7aa36f030ae4f37fb260.